serverless-plugin/dashboard.js

```diff
@@ -60,7 +58,7 @@ module.exports = function dashboard (serverless, config, context) {
       ...stateMachineWidgets,
       ...lambdaWidgets
     ])
-    const dash = { start: DASHBOARD_PERIOD, widgets: positionedWidgets }
+    const dash = { start: config.timeRange, end: config.timeRange.end, widgets: positionedWidgets }
```

Comment:
Shouldn't this be as follows? (see `.start`)

```suggestion
    const dash = { start: config.timeRange.start, end: config.timeRange.end, widgets: positionedWidgets }
```